src/Microsoft.SqlTools.ServiceLayer/ShowPlan/Contracts/ExecutionPlanGraph.cs

```diff
@@ -71,9 +71,9 @@ public class ExecutionPlanGraphElementProperties
         /// </summary>
```

Comment:
The name `ExecutionPlanGraphElementProperties` doesn't seem like an accurate name given how it seems to be used below - shouldn't it be something like `ExecutionPlanGraphProperty` instead?

Reply:
This property belongs to graph nodes and edges and not the graph itself. Will the new name still make sense?

Reply:
It's still a part of an ExecutionPlanGraph though so that still seems to make sense to me. But feel free to make it more generic if you want, I was mostly commenting on the `ElementProperties` part of the name